src/components/Form/Form.js

```diff
@@ -43,56 +44,78 @@ class Form extends Component {
-          placeholder='Scientific name'
-          name='scientific_name'
-          value={this.state.scientific_name}
-          onChange={e => this.handleChange(e)}
+        </div>
+        <div className="form-row">
+          <label htmlFor="scientificName">Scientific name</label>
+          <input
+            type='text'
+            id='scientificName'
+            name='scientific_name'
+            value={this.state.scientific_name}
+            onChange={e => this.handleChange(e)}
           />
-        <input
-          type='text'
-          placeholder='Region'
-          name='region'
-          value={this.state.region}
-          onChange={e => this.handleChange(e)}
+        </div>
+        <div className="form-row">
+          <label htmlFor="region">Region</label>
+          <input
+            type='text'
+            id='region'
+            name='region'
+            value={this.state.region}
+            onChange={e => this.handleChange(e)}
           />
-        <input
-          type='text'
-          placeholder='Average height'
-          name='average_height'
-          value={this.state.average_height}
-          onChange={e => this.handleChange(e)}
+        </div>
+        <div className="form-row">
+          <label htmlFor="averageHeight">Average height</label>
+          <input
+            type='text'
+            id='averageHeight'
+            name='average_height'
+            value={this.state.average_height}
+            onChange={e => this.handleChange(e)}
           />
-        <input
-          type='text'
-          placeholder='Lifespan'
-          name='lifespan'
-          value={this.state.lifespan}
-          onChange={e => this.handleChange(e)}
+        </div>
+        <div className="form-row">
+          <label htmlFor="lifespan">Lifespan</label>
+          <input
+            type='text'
+            id='lifespan'
+            name='lifespan'
+            value={this.state.lifespan}
+            onChange={e => this.handleChange(e)}
           />
-        <input
-          type='text'
-          placeholder='Fun fact'
-          name='fact'
-          value={this.state.fact}
-          onChange={e => this.handleChange(e)}
+        </div>
+        <div className="form-row">
+          <label htmlFor="funFact">Fun fact</label>
+          <input
+            type='text'
+            id='funFact'
+            name='fact'
+            value={this.state.fact}
+            onChange={e => this.handleChange(e)}
           />
-        <input
-          type='text'
-          placeholder='Image URL'
-          name='image'
-          value={this.state.image}
-          onChange={e => this.handleChange(e)}
+        </div>
+        <div className="form-row">
+          <label htmlFor="imageURL">Image URL</label>
+          <input
+            type='text'
+            id='imageURL'
+            name='image'
+            value={this.state.image}
+            onChange={e => this.handleChange(e)}
           />
```

Comment:
Just wondering what the ids on the inputs are used for? I didn't see them referenced anywhere and was wondering if maybe we could remove them.